src/test/unit_tests/test_nc_iam_basic_integration.js

```diff
@@ -229,6 +230,40 @@ mocha.describe('IAM basic integration tests - happy path', async function() {
             _check_status_code_ok(response);
         });
     });
+
+    mocha.describe('IAM other APIs (currently returns empty value)', async function() {
+        const username3 = 'Emi';
+
+        mocha.before(async () => {
+            // create a user
+            const input = {
+                UserName: username3
+            };
+            const command = new CreateUserCommand(input);
+            const response = await iam_account.send(command);
+            _check_status_code_ok(response);
+        });
+
+        mocha.after(async () => {
+            // delete a user
+            const input = {
+                UserName: username3
+            };
+            const command = new DeleteUserCommand(input);
+            const response = await iam_account.send(command);
+            _check_status_code_ok(response);
+        });
+
+        mocha.it('list groups for user - should be empty', async function() {
```

Comment:
I would add this check also in between of user create and delete.
Otherwise LGTM